webpack/environments/prodConfig.js

```diff
@@ -0,0 +1,33 @@
+// @ts-check
+const path = require("path")
+const WebpackManifestPlugin = require("webpack-manifest-plugin")
+const { HashedModuleIdsPlugin } = require("webpack")
+const TerserPlugin = require("terser-webpack-plugin")
+const { getCSSManifest } = require("../utils/getCSSManifest")
+const { NODE_ENV } = require("../../src/lib/environment")
+
+module.exports.prodConfig = {
+  parallelism: 75,
+  mode: NODE_ENV,
+  devtool: "source-map",
+  output: {
+    filename: "[name].[contenthash].js",
+  },
+  plugins: [
+    new HashedModuleIdsPlugin(),
+    new WebpackManifestPlugin({
+      fileName: path.resolve(__dirname, "../../manifest.json"),
+      basePath: "/assets/",
+      seed: getCSSManifest(),
+    }),
+  ],
+  optimization: {
+    minimizer: [
+      new TerserPlugin({
```

Comment:
Again, it _may_ be sufficient to just fallback to the defaults. I'm not 100% sure what the default configuration is though, so that might be worth double checking.